csharp/compatibility_tests/v3.0.0/src/Google.Protobuf.Test/Google.Protobuf.Test.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net451;netcoreapp1.0</TargetFrameworks>
+    <TargetFrameworks>net451;netcoreapp2.2</TargetFrameworks>
```

Comment:
I'd prefer to use netcoreapp2.1 instead of netcoreapp2.2 because that corresponds to an LTS version of .NET core

https://dotnet.microsoft.com/platform/support/policy/dotnet-core

Reply:
Just compatibility tests or all projects?

Reply:
I think all .exe project - otherwise we'd need to install yet another SDK.